src/components/developer-website/questionContainer/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { withTheme } from "styled-components";
+import { MainHeader, OrangeButton } from "./style.js";
```

Comment:
Super minor but I just want this to be consistent: should be `styles.js` instead of `style.js` 👍